ckan/templates/snippets/activity_stream.html

```diff
@@ -0,0 +1,51 @@
+{% macro actor(activity) %}
+<span class="actor">
+  {{ h.linked_user(activity.user_id, 0, 30) }}
+</span>
+{% endmacro %}
+
+{% macro dataset(activity) %}
+<span class="dataset">
+  {{ h.dataset_link(activity.data.package) }}
+</span>
+{% endmacro %}
+
+{% macro organization(activity) %}
+{{ h.organization_link(activity.data.group) }}
```

Comment:
If I'm not mistaken this uses the Org / group name, not the title. Would it be possible to use the title? Datasets seem to use the title. And I understand that this is very likely how it was implemented before your changes, just wondering